src/anomalib/pipelines/components/base/pipeline.py

```diff
@@ -66,9 +66,9 @@ def run(self, args: Namespace | None = None) -> None:
             except Exception:  # noqa: PERF203 catch all exception and allow try-catch in loop
                 logger.exception("An error occurred when running the runner.")
                 print(
-                    f"There were some errors when running [red]{runner.generator.job_class.name}[/red] with"
-                    f" [green]{runner.__class__.__name__}[/green]."
-                    f" Please check [magenta]{log_file}[/magenta] for more details.",
+                    f"There were some errors when running {runner.generator.job_class.name} with"
```

Comment:
should we use print or just log it ?

Reply:
Since the logs are being redirected to a file, the idea was to also inform the users in the terminal that an error has occurred, and that if they want to see the complete detail they can refer to the file.

Reply:
ah right, thanks for the clarification